src/Moryx.TestTools.UnitTest/DummyLogger.cs

```diff
@@ -138,6 +116,11 @@ public IModuleLogger GetChild(string name, Type target)
         {
             return new DummyLogger();
         }
+
+        public void SetNotificationTarget(Action<LogLevel, string, Exception> notificationTarget)
+        {
+            throw new NotImplementedException();
```

Comment:
What was your intention here? Do we still need the DummyLogger test class, if yes should there be an implementation here?

Reply:
Since we still offer the `ModuleLogger`, we also need to provide the `DummyLogger` for those cases.